src/styles/issues.scss

```diff
@@ -115,6 +115,8 @@
   }
   .issues-filter-wrapper {
     display: none;
+    max-height: 800px;
```

Comment:
Using px isn’t responsive, could you turn this to vh?

Reply:
okay

Reply:
@alexdor changing to vh does not make the sidebar scrollable its just fixed and you cannot it scroll it then

Reply:
You have to adjust the value as well, vh is a percentage of the whole viewport. So when you say 800vh for example you mean 8 times the viewport height. You need a value around 80 probably

Reply:
Ya sorry my fault